priv/repo/migrations/20240303150357_create_shared_programs.exs

```diff
@@ -0,0 +1,12 @@
+defmodule Cadet.Repo.Migrations.CreateSharedPrograms do
+  use Ecto.Migration
+
+  def change do
+    create table(:shared_programs) do
+      add(:uuid, :uuid)
+      add(:data, :map)
```

Comment:
Can we add constraints and/or indices to this table (e.g. non-null, unique, primary key, etc.)?